{{cookiecutter.project_slug}}/clients/web/vue3/src/views/ResetPassword.vue

```diff
@@ -58,7 +58,7 @@ export default {
 
       const { uid, token } = route.params
 
-      User.api
+      userApi.csc
```

Comment:
what's `csc`?

Reply:
It's just an alias for `customServiceCall`

https://github.com/thinknimble/tn-models-fp#createcustomservicecall

`.customServiceCall` can still be used if you preferred the longer name.

Ideally all method calls should probably be available at the root of the api object, but the typing for that was quite a pain so any custom call live under `customServiceCall`.